packages/terra-form-select/src/Combobox.jsx

```diff
@@ -110,20 +110,14 @@ const defaultProps = {
 };
 
 class Combobox extends React.Component {
-  static defaultValue(props) {
-    if (props.value !== undefined) {
-      return null;
-    }
-
-    return (props.defaultValue !== undefined && props.defaultValue !== null) ? props.defaultValue : '';
-  }
-
   constructor(props) {
     super(props);
 
+    const { value, defaultValue } = props;
+
     this.state = {
       tags: [],
-      value: Combobox.defaultValue(props),
+      value: SelectUtil.defaultValue({ defaultValue, value }),
```

Comment:
I'm wondering if it would have been possible to scope this change only within the Select component.

Rough Pseudo code
```
const { variant,  value, defaultValue, ...otherProps } = props;

const passiveDefaultValue = SelectUtil.defaultValue({ defaultValue, value }); 

switch (variant) {
  case: Variants.Variant:
    return <Variant defaultValue={passiveDefaultValue} ...otherProps />;
}
```

Mainly because the variants themselves are new, so it is completely passive to keep these strict prop-types if a user was to import them directly. `import Combobox from 'terra-form-select/lib/Combobox'`. It is only non-passive when using the base Select as the entry point.

Reply:
It gets tricky for sure, I think the lists are a single value. `[value]`. I'm not really sure what would happen if a list of two or more items were provided to a single select variant. `[value1, value2]`. Chaos, I assume. I think your current solution satisfies the issue, it's just unfortunate we have to expose new prop types to specialized components.

We could probably transform the value in a very similar way to the defaultValue, but it's probably not worth the risk. Controlled values should probably just pass through.

Reply:
Yeah, it's just such an odd use case... And it doesn't really work, see below:

![image](https://user-images.githubusercontent.com/34115417/65055600-19bfb900-d935-11e9-86d7-8debfb30442f.png)

Same behavior whether we use `value` or `defaultValue` here.

I thought about logging an issue to circle back and remove the weird list prop-types for the single selects for the next breaking change

Reply:
+1 to circling back around on the next major version

Reply:
Issue logged: https://github.com/cerner/terra-core/issues/2652